lib/power_action_utils.pm

```diff
@@ -220,30 +221,29 @@ sub power_action {
             }
         }
     }
+    my $soft_fail_data;
     # Shutdown takes longer than 60 seconds on SLE 15
     my $shutdown_timeout = 60;
     if (is_sle('15+') && check_var('DESKTOP', 'gnome') && ($action eq 'poweroff')) {
-        record_soft_failure('bsc#1055462');
-        $shutdown_timeout *= 3;
+        $soft_fail_data = {bugref => 'bsc#1055462', soft_timeout => 60, timeout => 180};
     }
     # The timeout is increased as shutdown takes longer on Live CD
     if (get_var('LIVECD')) {
-        $shutdown_timeout *= 4;
+        $soft_fail_data = {bugref => 'bsc#1096241', soft_timeout => 60, timeout => 300};
     }
     if (get_var("OFW") && check_var('DISTRI', 'opensuse') && check_var('DESKTOP', 'gnome') && get_var('PUBLISH_HDD_1')) {
-        $shutdown_timeout *= 3;
-        record_soft_failure("boo#1057637 shutdown_timeout increased to $shutdown_timeout (s) expecting to complete.");
+        $soft_fail_data = {bugref => 'bsc#1057637', soft_timeout => 60, timeout => 180};
     }
     # no need to redefine the system when we boot from an existing qcow image
     # Do not redefine if autoyast or s390 zKVM reboot, as did initial reboot already
     if (check_var('VIRSH_VMM_FAMILY', 'kvm')
         || check_var('VIRSH_VMM_FAMILY', 'xen')
         || (get_var('S390_ZKVM') && !get_var('BOOT_HDD_IMAGE') && !get_var('AUTOYAST') && $args{first_reboot}))
     {
-        assert_shutdown_and_restore_system($action, $shutdown_timeout);
+        assert_shutdown_and_restore_system($action);
```

Comment:
Hmm, so here we won't expect shutdown to take longer ever? Previously we would have increased timeout here in certain cases.